markdown/dev/guides/writers/files/en.md

```diff
@@ -0,0 +1,41 @@
+---
+title: File locations
+order: 200
+---
+
+With the exception of blog posts, showcase posts, and newsletters,
+all of our web pages can be found in our repository on GitHub.
+
+## Design documentation
+
+Documentation for our designs are found in the `/markdown/org/docs/patterns` directory.
```

Comment:
Note that `patterns` has been renamed to `designs` so the documentation for designs is in `markdown/org/docs/designs` now